api/test/e2enew/route/route_with_plugin_orchestration_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/extensions/table"
 	"github.com/onsi/gomega"
+	"github.com/tidwall/gjson"
 
```

Comment:
Hi, this file seems doesn't test `script` field?

Reply:
The request body was contained in dag-conf.json.Also include the `script` field.
You can see this file line 32.

```
bytes, err := ioutil.ReadFile("../../testdata/dag-conf.json")
```